### ISSUE TITLE
PDB2PQR should not crash when reading malformed PDB headers

### ISSUE THREAD

**Issue by sobolevnrm** [Electrostatics/pdb2pqr#210]
The code should be more tolerant of the odd `REMARK` and similar headers that some files generate.

**Comment by sobolevnrm** [comment]
For example:
```
REMARK    GENERATED BY TRJCONV
REMARK    THIS IS A SIMULATION BOX
```

**Comment by Eo300** [comment]
From what I found, this appears to be already be addressed in the `master` branch in `pdb.py`:
https://github.com/Electrostatics/pdb2pqr/blob/1264a73e7a62f747af4b8cd09fa0dba02ddb1473/pdb2pqr/pdb.py#L2581-L2590

Where it's eventually detected in io.py:
https://github.com/Electrostatics/pdb2pqr/blob/1264a73e7a62f747af4b8cd09fa0dba02ddb1473/pdb2pqr/io.py#L462-L467

However, the former snippet was added after v3.1.0 was released, hence the latest release crashing on malformed REMARKS

**Comment by sobolevnrm** [comment]
Fixed in 8c810ca64e6a6fb4c39c2394f3da28d317fc6350
